community/urlview/depends

```diff
@@ -0,0 +1,5 @@
+autoconf make
+automake make
+gzip make
+ncurses
+patch make
```

Comment:
Remove this.

Reply:
the patch fails with `busybox patch`

Reply:
Fix the patch (if it really is required).